modules/core/include/opencv2/core/opencl/opencl_info.hpp

```diff
@@ -47,6 +47,22 @@ static std::string bytesToStringRepr(size_t value)
         s = s.substr(0, s.size() - 1);
     return s;
 }
+
+static String getDeviceTypeString(const cv::ocl::Device& device) {
```

Comment:
Braces should go on a new line. Or you can drop them entirely when they are unnecessary. 
Otherwise, the code looks good to me.

Reply:
Again, correct. Actually, does opencv have an autoformat file (clang-format) that I can apply?

Reply:
Some coding style information is available on Wiki: https://github.com/opencv/opencv/wiki/Coding_Style_Guide#code-layout

`clang-format` looks weird (introduces too many unnecessary changes - there is no way to specify "keep"/"do not care" option for some rules) - you may find PR with these experiments.

Reply:
Got it, thanks a lot for the info!